### ISSUE TITLE
UNO WIFI REV2 Schematic does not match hardware

### ISSUE THREAD

**Issue by TimEllis2019** [arduino/Arduino#8383]
The schematics UNOWiFiRev2v4.0 15/10/18 do not match the hardware I was sent.  The schematics list LED_BUILTIN as being on pin 26 of the MEGA4809 but the led is driven from pin 25.  Where can I get a correct set of schematics?  Is there a file listing the pin numbers from the processor and the connectors on the board i.e. digital 0 - 13?
-tim

**Comment by per1234** [comment]
>When the documentation for pinMode(pin, type) refers to a pin the pin is the pin on the edge connector not the chip? The compiler does a translation based on the kind of microprocessor that is selected for compile?

That is correct. Every official Arduino API takes Arduino pin numbers (as marked on the silkscreen of the board) and pretty much all the 3rd party Arduino libraries do the same.

>the documentation for pinMode, digitalWrite and digitalRead should clarify this.

I'll consider how the documentation might be improved to make this more clear.
